### DISCUSSION TITLE
Every example in Javascript

### DISCUSSION THREAD

**Discussion by luglio7** [lit/lit#2101] [Ideas]
Typescript is cool, Typescript is fantastic .... but the language of the web is Javascript and is very hard to read the documentation for a Javascript developer since every example is in Typescript.

**Comment by abdonrd** [comment]
Seems like they are working on it.

Related:
- https://github.com/lit/lit.dev/issues/332
- https://github.com/lit/lit/pull/1919

**Reply by luglio7** [reply]
thanks

**Reply by graynorton** [reply]
Yup—this has been in the works for a while and will land soon.

**Comment by e111077** [comment]
It has landed on the site by the way. Work is still being done to handle the parts that are not in the code examples that are TS-oriented.

**Reply by aomarks** [reply]
I think all the text is JS/TS neutral now too -- except the parts that are explicitly about TypeScript. Should file a bug if something is not though.
